README.md

```diff
@@ -20,6 +20,49 @@ HOWEVER.  Let me dispel your concerns with this: LGNC does support HTTP transpor
 * in `POST` requests data is always sent in body as JSON or MsgPack
 * SSL must be terminated on reverse-proxy level (nginx etc.)
 
+### tl;dr
+
+LGNC is not a framework, but a convention on sending requests over networks (not just HTTP): it defines
+request/response envelope format, ensures that all participants know signatures of all contracts
+(i.e. no blackboxes/gentlemen's agreements), and guarantees that assertions above will be rock hard
+regardless of concrete language implementation.
+
+### Use cases
+
+There are two main use cases for LGNC:
+
+1. iOS app + backend: say you have a mobile application<sup id="a1">[*](#f1)</sup>, and most probably you would want
+it to communicate with your backend. Usually it's a blackbox backend on PHP/Python/Java/.NET/Go, and you send
+requests to it using Alamofire/URLSession/Swift-NIO. The problem here is that your contract with backend may break
+any time, not to mention that you have to map every request/response to a struct/class, and sometimes response
+might be in a slightly different format (remember that "do we send booleans as true/false or 0/1?" discussion).
+LGNC allows you to just dodge all that problems, proceeding directly to business tasks, all you have to negotiate
+with backend team is a contract signature in clean and simple YAML format, after that you generate boilerplate
+contracts for your app and execute them directly (as code, not JSON or whatever), without having to bother about
+transport or format details, and in the meantime BE team does precisely the same thing: they generate boilerplate
+contracts for BE, guarantee them and just hook up business logic to it. You will never catch an "oof, I expected
+this field to be integer, and it actually is, but for some reason in quotes, and because of that Codable failed
```

Comment:
```suggestion
this field to be integer, and it actually is, but for some reason it is in quotes, and because of that Codable failed
```
Otherwise could be read as "reason in quotes"